azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -500,8 +503,7 @@ private static WebAppConfiguration getConfigurationFromExisting(IWebApp webapp,
             if (StringUtils.isNotBlank(imageSetting)) {
                 builder.image(imageSetting);
             } else {
-                // TODO: docker support in appservice-lib
-                // builder.image(getDockerImageName(webapp.linuxFxVersion()));
+                builder.image(webapp.entity().getDockerImageName());
```

Comment:
`imageSetting` may contains the reigstry url, may also need to handle this case. Can we use `webapp.entity().getDockerImageName()` directly?

Reply:
No need by now